common/upload.py

```diff
@@ -40,23 +41,44 @@ def check_file_type(self, file, type):
 class ZipUploader(Uploader):
     def __init__(self, file):
         super().__init__()
-        self.archive = zipfile.ZipFile(file, "r")
+        self.archive = self.archive = zipfile.ZipFile(file,"r")
```

Comment:
Duplicated assignment.
```suggestion
        self.archive = zipfile.ZipFile(file, "r")
```